Helper/Parser.php

```diff
@@ -36,6 +36,10 @@ public function parseText($text, $checkTable = true) {
     foreach ($matches[0] as $match) {
       $text = str_replace($match, $this->htmlCodes[$match], $text);
     }
+    $text = str_replace("\\", "\\backslash ", $text);
+    $text = str_replace("{", "\\{ ", $text);
+    $text = str_replace("}", "\\} ", $text);
```

Comment:
By merging these (\,{,}) changes, we drop the support for native latex-commands, which is something I do not want to do. We should probably add an option to enable stripping the latex-commands (which enables these lines) in the generator.